resources/settings.json

```diff
@@ -3,20 +3,20 @@
     // The bind address and port of the lobby host.
```

Comment:
Regent added the comments, so might wanna ask him if you wanna remove them

Reply:
Ah fuck, this might be vscode formatter being evil, i dont remember removing these comments.

Reply:
Ok this is stupid. They're not gone in my local repo. It's github that's being evil